pkgs/development/libraries/libpam-wrapper/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitLab
+, cmake
+, openpam
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  version = "1.1.3";
+  pname = "libpam-wrapper";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.com";
```

Comment:
This is the default.

Reply:
Actually, I see no mention of GitLab on the website: https://cwrap.org/download.html#pam_wrapper